internal/pkg/cli/prompt/interactive/interactive_test.go

```diff
@@ -213,12 +213,11 @@ func TestPrompt_HideLeaveBlank(t *testing.T) {
 
 		assert.NoError(t, console.ExpectString("My input"))
 
-		err = console.ExpectString("Leave blank for default value.")
-		assert.Error(t, err)
-
 		assert.NoError(t, console.SendEnter()) // enter - default value
 
 		assert.NoError(t, console.ExpectEOF())
+
+		assert.NotContains(t, console.String(), "Leave blank for default value.")
```

Comment:
Tu testujeme, ze dana hlaska sa vo vystupe neobjvi, zbytocne sa caka `5s` kym nastane timeout.
Je lepsie to skontrolovat na zaver.